cli/cmd/middleware.go

```diff
@@ -84,8 +82,8 @@ func (p *resourceParameters) Validate(cmd *cobra.Command, args []string) []error
 
 	p.ResourceName = args[0]
 
-	_, err := resources.Get(p.ResourceName)
-	if errors.Is(err, resourcemanager.ErrResourceNotFound) {
+	exists := resources.Exists(p.ResourceName)
```

Comment:
I changed this to use a different function because `Get` now writes the warning to the console if deprecated